public/app/features/alerting/unified/Insights.tsx

```diff
@@ -0,0 +1,64 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { EmbeddedScene, SceneFlexLayout, SceneTimeRange } from '@grafana/scenes';
+import { useStyles2 } from '@grafana/ui';
+
+import { getFiringAlertsScene } from './insights/grafana/FiringAlertsPercentage';
+import { getFiringAlertsRateScene } from './insights/grafana/FiringAlertsRate';
+import { getMostFiredInstancesScene } from './insights/grafana/MostFiredInstancesTable';
+import { getMostFiredRulesScene } from './insights/grafana/MostFiredRulesTable';
+import { useAlertmanager } from './state/AlertmanagerContext';
+import { GRAFANA_RULES_SOURCE_NAME } from './utils/datasource';
+
+//all cloud instances are guaranteed to have this datasource uid for the alert state history loki datasource
```

Comment:
What about OSS self-hosted instances? We should probably get this information from the config

Reply:
Agree although we decided to initially focus only on cloud customers